src/app/rundown/components/rundown-header/rundown-header.component.ts

```diff
@@ -102,15 +108,19 @@ export class RundownHeaderComponent implements OnInit, OnDestroy, OnChanges {
   }
 
   private isDesignInfinitePiece(piece: Piece): boolean {
-    return piece.name.startsWith('DESIGN_')
+    return piece.name.startsWith(DESIGN_TEMPLATE_IDENTIFIER)
   }
 
   private isSkemaInfinitePiece(piece: Piece): boolean {
-    return piece.name.startsWith('DESIGN_')
+    return piece.name.startsWith(SKEMA_TEMPLATE_IDENTIFIER)
   }
 
   private getGfxNameFromTemplate(template: string) {
     const pattern: RegExp = /^.+_(?<gfxName>\w+)$/
-    return template.match(pattern)?.groups?.['gfxName'] ?? template
+    const { gfxName } = template.match(pattern)?.groups ?? {}
+    return gfxName ?? template
   }
+
+  protected readonly IconButton = IconButton;
+  protected readonly IconButtonSize = IconButtonSize;
```

Comment:
Please place instance variables at the top of the class. This is for consistency between all our classes, so if we ever want to check the instance variables of a class, there is only one place for us to check.

Sidenote: What's the reason for making them protected?

Reply:
`protected` makes them available to the HTML template, but not outside components.

Reply:
Good catch, i missed these. They were auto generated at some point, and they are automatically put there at the bottom. I assume they are protected to limit exposure, but being private wouldn't let them be recognized by the AOT compiler.

Reply:
I have moved them now!